examples/lib/defaultHtmlStepUi.js

```diff
@@ -45,7 +45,7 @@ function DefaultHtmlStepUi(_sequencer, options) {
                 <div class="col-md-8 cal collapse in step-column">\
                   <div class="load load-spin" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></div>\
                   <div class="step-image">\
-                    <a class="cal collapse in"><img class="img-thumbnail step-thumbnail"/></a>\
+                    <a class="cal collapse in"><img id="StepImageElement" class="img-thumbnail step-thumbnail"/></a>\
```

Comment:
Giving an id would lead to having more than one elem with the same id on page. I think this should be changed to a class.

Reply:
I did try it with class but the functions require using and img id! So that is why id was added.

Reply:
![id-poss](https://user-images.githubusercontent.com/26546120/59975047-4cc20d00-95d1-11e9-816b-887010f716a9.png)

Reply:
Can you directly provide a DOM element to the function?

Reply:
no it asks for a canvas or image id .

Reply:
So the image cannot be given as a DOM element? This will create problems with ids.

Reply:
Oh wait
you do `var imgToProcess = document.getElementById('StepImageElement');` this

You can just get the element using a class. getElementsByClass or querySelector.

Reply:
I did try doing these as I mentioned and I tried again it throws error. I think the approach used works fine!

Reply:
but it will only pick the first img on the page. Have you tried adding multiple steps? like try adding blob-analysis then import a fresh image(same one) and run blob analysis again.

Reply:
okay I ll try something else and see if it can be used!

Reply:
Thanks!